tests/main/snap-get/task.yaml

```diff
@@ -1,5 +1,15 @@
 summary: Check that `snap get` works as expected
 
+details: |
+    Certain snaps, such as those providing a background service, expose
```

Comment:
Configuration works for all snaps, not only services.

Reply:
Right, I got this from the official documentation :)
I'll update the detail